cardano-cli/golden/files/golden/help/drep_registration-certificate.cli

```diff
@@ -0,0 +1,28 @@
+Usage: cardano-cli drep registration-certificate 
+            (--mainnet | --testnet-magic NATURAL)
+            ( --drep-verification-key STRING
+            | --drep-cold-verification-key-file FILE
+            )
+            (--vrf-verification-key STRING | --vrf-verification-key-file FILE)
+            [--metadata-url URL --metadata-hash HASH]
+            --out-file FILE
+
+  Create a DRep registration certificate
+
+Available options:
+  --mainnet                Use the mainnet magic id. This overrides the
+                           CARDANO_NODE_NETWORK_ID environment variable
+  --testnet-magic NATURAL  Specify a testnet magic id. This overrides the
+                           CARDANO_NODE_NETWORK_ID environment variable
+  --drep-verification-key STRING
+                           DRep verification key (Bech32 or hex-encoded).
+  --drep-cold-verification-key-file FILE
+                           Filepath of the drep verification key.
+  --vrf-verification-key STRING
+                           VRF verification key (Bech32 or hex-encoded).
+  --vrf-verification-key-file FILE
+                           Filepath of the VRF verification key.
+  --metadata-url URL       Pool metadata URL (maximum length of 64 characters).
+  --metadata-hash HASH     Pool metadata hash.
+  --out-file FILE          The output file.
```

Comment:
We are missing the deposit. 

https://github.com/cardano-foundation/CIPs/blob/005b80e42a27c69ba92f671c0a681905ae7b1a34/CIP-1694/README.md?plain=1#LL393C1-L395C12